gitstats

```diff
@@ -389,6 +390,11 @@ class GitDataCollector(DataCollector):
 
             sha1, fullpath = line.split()[2:4]
 
+            # skip directories we aren't interested in
+            if len(conf['exclude_directories']):
+                if any(dir in fullpath for dir in conf['exclude_directories'].split(',')):
```

Comment:
`dir` is a keyword in Python. This should be changed to something like `path`.

There's no need to check the length -- if you iterate over the `.split()` results nothing will happen if it's empty. However, this should be a list or a tuple instead. I know the `exclude_extensions` value I hard-coded is comma-separated, but I should change that.

There should be a way to pass in directories to exclude at the command line. Or possibly with a config file, such as `.bashrc` or `.vimrc`. I'm open to either one -- preferably both. There's a TODO in there to replace getopt with argparse, so if you're feeling really ambitious you could do that.